app/config/errors.php

```diff
@@ -10,61 +10,64 @@
     /** General Errors */
     Exception::GENERAL_UNKNOWN => [
         'name' => Exception::GENERAL_UNKNOWN,
-        'description' => 'An unknown error has occured. Please check the logs for more information.',
+        'description' => 'An unknown error has occured. Please submit a support ticket at [TODO: Where do people get help?]() with reproduction steps for further assistance.',
         'code' => 500,
     ],
     Exception::GENERAL_MOCK => [
         'name' => Exception::GENERAL_MOCK,
-        'description' => 'General errors thrown by the mock controller used for testing.',
+        'description' => 'General errors thrown by the mock controller used for testing. If you\'re not building Appwrite from source, please raise an issue at [TODO: Where do people get help?]()',
         'code' => 400,
     ],
     Exception::GENERAL_ACCESS_FORBIDDEN => [
         'name' => Exception::GENERAL_ACCESS_FORBIDDEN,
-        'description' => 'Access to this API is forbidden.',
+        'description' => 'Access to this API is forbidden. Please [action to get access]',
```

Comment:
When is this error API ever used? Why would access be forbidden? What are practical next steps here?

Reply:
Theoretically, this happens when someone tries to access the console or project APIs, but specifies a different project:

https://github.com/appwrite/appwrite/blob/27adf0788ff9809b9e3853480e498e5e169c81b2/app/controllers/api/console.php#L13

https://github.com/appwrite/appwrite/blob/27adf0788ff9809b9e3853480e498e5e169c81b2/app/controllers/api/projects.php#L45

We can say to please use `console` as the project for this request.

Reply:
That is better, I think for "contributor devex" this is important still. Thanks